### DISCUSSION TITLE
Allow to cache cold start caching of apps to nx cloud

### DISCUSSION THREAD

**Discussion by mykhailo-zakharov** [nrwl/nx#23277] [Feature Requests]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Description
I'd like to have a cache of cold start when I use nx cloud solution
the issue is the `npm run start` command takes 6 minutes for the large app. 

## Motivation
more people will want to use nx cloud solution

## Suggested Implementation
it's possible to configure webpack to save cache file to disc and then save it to nx cloud 

## Alternate Implementations
work with Vite, other bundlers

**Comment by FrozenPandaz** [comment]
Converting this to a discussion so that it can be discussed more easily.

Are you referring to running `npm run start` locally or as part of another task such as running `e2e` tests in CI?

**Reply by mykhailo-zakharov** [reply]
I mean locally running the script 
Maybe the solution should more come from bundle package like webpack
I just want to hear some thoughts about it
